nss-day-cohort-52/giffygram-nunchuck-scorpions
# Create Post Feature Implementation

#### Changes Made

1. Modified files provider.js, PostEntry.js, and GiffyGram.js
2. in GiffyGram.js I added an event listener for the create post button, as well as in if statement in the function body of GiffyGram
3. added a posts resource in application state in provider.js, as well as a sendPost function
4. in PostEntry.js I added an event listener for the submitPost button, this stores the inputs from the user, and makes them into an object to be sent to the sendPost function, as well as clears the sessionStorage 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cs-createpost
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    please make sure all features are working, to do this first click on "have a gif?" button, make sure the form populates. then test functionality of the submit post button (doesn't render on page yet, go to database and see if it was populated there, also keep in mind there is no error prevention in the text fields, so please only enter things that make sense)
Also check the functionality of cancel post button, when clicked should remove the create post form from the dom until you click on the "have a gif?" button again
4. View code file.
    view files in provider.js, PostEntry.js, and GiffyGram.js
